src/app/qgisapp.cpp

```diff
@@ -6852,9 +6852,9 @@ void QgisApp::saveAsRasterFile( QgsRasterLayer *rasterLayer )
     if ( rlWeakPointer )
       emit layerSavedAs( rlWeakPointer, fileName );
 
-    messageBar()->pushMessage( tr( "Saving done" ),
-                               tr( "Export to raster file has been completed" ),
-                               QgsMessageBar::INFO, messageTimeout() );
+    messageBar()->pushMessage( tr( "Saving Done" ),
+                               tr( "Successfully saved raster layer to <a href=\"%1\">%2</a>" ).arg( QUrl::fromLocalFile( QFileInfo( newFilename ).path() ).toString() ).arg( newFilename ),
```

Comment:
I realise we need to do this everywhere, but we should add QDir::toNativeSeparators here:

    .arg( QDir::toNativeSeparators( newFilename ) )

Reply:
@nyalldawson , OK, I'll change it everywhere.